### PR TITLE
unarchive service discovery tests

### PR DESCRIPTION
**Pull request by bloom1**

### CODE REVIEW

File: wazo_acceptance/steps/consul.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+                     'host': context.wazo_config['wazo_host'],
+                     'port': 8500,
+                     'verify': False}
+    finder = ServiceFinder(consul_config)
+
+    def test():
+        healthy_services = finder.list_healthy_services(service_name)
+        assert_that(healthy_services, not_(empty()))
+
+    until.assert_(test, tries=5)
```
Comment (sduthil) [inline wazo_acceptance/steps/consul.py]: Please use `timeout=` instead of `tries=`, it is clearer.

File: wazo_acceptance/steps/consul.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+@then('consul returns a running "{service_name}" service')
+def consul_returns_a_running_service(context, service_name):
+    consul_config = {'token': context.wazo_config['consul_token'],
+                     'scheme': 'https',
+                     'host': context.wazo_config['wazo_host'],
+                     'port': 8500,
+                     'verify': False}
+    finder = ServiceFinder(consul_config)
+
+    def test():
```
Comment (sduthil) [inline wazo_acceptance/steps/consul.py]:
```suggestion
    def service_is_healthy(service_name):
```

File: wazo_acceptance/steps/consul.py (hunk 1 above)
Comment (sduthil) [inline wazo_acceptance/steps/consul.py]:
```suggestion
    until.assert_(service_is_healthy, service_name, timeout=5)
```

Comment (bloom1) [reply wazo_acceptance/steps/consul.py]: All fixed.

**Comment by fblackburn1** [comment]
Can you retry tests on buster

**Pull request merged**